core/services/ocr2/plugins/ocr2keeper/integration_21_test.go

```diff
@@ -200,7 +200,7 @@ func TestIntegration_KeeperPluginLogUpkeep(t *testing.T) {
 	nodes := setupNodes(t, nodeKeys, registry, backend, steve)
 	// wait for nodes to start
 	// TODO: find a better way to do this
-	<-time.After(time.Second * 10)
+	blockWait(10, backend)
```

Comment:
I am not sure if this is a good way and might still flake in CI. Can we try to look at other tests in the repo how they try to verify nodes have started up? 

One way I can think of is to check logs on the node for particular log but not sure if that's even possible to do

Reply:
Do we know what that log should look like?

I've tried reading from node.app.Started, and also checking if the Server.URL is not empty, but neither is sufficient as a "is this node started?" check. There were also some functions like `IsReady` and `IsHealthy` but again, they didn't serve as a useful check. 

I've tried creating a new logger that wraps the SugaredLogger, and can check the log message that it receives, but from running the code, I only get a handful of Debug/Debugf/Debugw messages, and I was expecting a lot more at that level? 

Do you know what message in particular, and at what log level, we should be checking for to signal a node as ready?